tensorboard/webapp/widgets/data_table/filter_dialogue.ng.html

```diff
@@ -0,0 +1,64 @@
+<!--
+@license
+Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<div class="filter-dialogue">
+  <div
+    *ngIf="filter.type === DomainType.DISCRETE"
+    class="discrete-filters-area"
+  >
+    <tb-filter-input
+      *ngIf="filter.possibleValues"
+      [value]="discreteValueFilter"
+      (keyup)="discreteValueKeyUp($event)"
+      placeholder="Filter Discrete Values (regex)"
+    ></tb-filter-input>
+    <div *ngIf="!getPossibleValues().length" class="no-matches">
```

Comment:
I have some minor concerns around running this filter twice on every render if there gets to be a large number of options but I would like to find a way to test that situation first before spending the time to optimize this.